static/js/app.js

```diff
@@ -412,3 +412,16 @@ downloadPlaybookBttn?.addEventListener('click', () => {
     downloadDropdown.classList.add('hidden');
   });
 })
+
+// Initialize Embla carousel
```

Comment:
The JS here comes from Embla's [docs](https://www.embla-carousel.com/guides/previous-and-next-buttons/).  It is possible to include an autoplay, but I noticed that if a user hovers over a testimonial the feature will stop and there's no way for a user to resume autoplay without adding some custom JS. If we'd like to include autoplay, that can be handled in a future story.

Reply:
Glad to see you considered this and went beyond the initial ask for this story.  Definitely agreed that the extra functionality can be handled in the future.